pkg/metrics/metricfactory/metric_factory.go

```diff
@@ -101,15 +101,6 @@ func QATMetricsPromDesc(context string) (descriptions map[string]*prometheus.Des
 	return descriptions
 }
 
-func NodeCPUFrequencyMetricsPromDesc(context string) (descriptions map[string]*prometheus.Desc) {
```

Comment:
FYI: this needs further clean up in manifests/compose/mock-acpi/grafana/dashboards/mock-acpi/dashboard.json 

cc: @vimalk78